### PR TITLE
Add possibility to create KSCrash instance with custom reports directory

### PR DESCRIPTION
**Pull request by BamX**

### CODE REVIEW

**Comment by yvasileuski** [comment]
Cool, it will help us to separate two KSCrash versions in the same app.

File: Source/KSCrash/Recording/KSCrash.m [hunk 1]
```diff
@@ -117,6 +117,11 @@ @implementation KSCrash
 
 - (id) init
 {
+    return [self initWithReportFilesDirectory:KSCRASH_ReportFilesDirectory];
```
Comment (kstenerud) [inline Source/KSCrash/Recording/KSCrash.m]: Since this is now a default, let's call it `KSCRASH_DefaultReportFilesDirectory`

Comment (BamX) [inline Source/KSCrash/Recording/KSCrash.m]: Ok, done

**Comment by kstenerud** [comment]
Awesome! Thanks!

**Pull request merged**